release_docs/INSTALL_cross_compile.txt

```diff
@@ -0,0 +1,66 @@
+
+***********************************************************************
+*       HDF5 Build and Install Instructions for Cross Compiling       *
+***********************************************************************
+
+Cross-compiling has several consequences for CMake:
+    CMake cannot automatically detect the target platform.
+    CMake cannot find libraries and headers in the default system directories.
+    Executables built during cross compiling cannot be executed.
```

Comment:
Would  this be more accurate?
Executables built during cross compiling can be executed only on the target platform.